Resources/Prototypes/Entities/Structures/Wallmounts/Signs/flags.yml

```diff
@@ -8,6 +8,23 @@
   - type: Sprite
     drawdepth: WallMountedItems
     sprite: Structures/Wallmounts/flags.rsi
+  - type: Destructible # DeltaV - Craftable Pride Flags
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 25
```

Comment:
put the type on the same line as trigger
```yml
- trigger: !type:DamageTrigger
  damage: 25
```

also add a higher trigger at say 50 that just deletes it for it to not cause lag when nuked
see wall/girder for an example of it